tests/unit/_internal/test_store.py

```diff
@@ -12,6 +13,11 @@
 from bentoml._internal.store import StoreItem
 from bentoml._internal.types import Tag
 
+if sys.version_info < (3, 7):
+    from backports.datetime_fromisoformat import MonkeyPatch
```

Comment:
@sauyon

Reply:
Seems fine to me?

Reply:
I will merge this in then

Reply:
We will probably have to wait from users feedback if there are any performance issue